ddtrace/internal/telemetry/metrics_namespaces.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any
+from typing import Dict
+
+from ddtrace.internal.constants import TELEMETRY_APPSEC
+from ddtrace.internal.constants import TELEMETRY_TRACER
+from ddtrace.internal.telemetry.metrics import CountMetric
+from ddtrace.internal.telemetry.metrics import GaugeMetric
+from ddtrace.internal.telemetry.metrics import Metric
+from ddtrace.internal.telemetry.metrics import MetricTagType
+from ddtrace.internal.telemetry.metrics import MetricType
+from ddtrace.internal.telemetry.metrics import RateMetric
+
+
+NamespaceMetricType = Dict[str, Dict[str, Any]]
+
+
+class TelemetryTypeError(Exception):
+    pass
+
+
+class MetricNamespace:
+    _metrics_data = {TELEMETRY_TRACER: {}, TELEMETRY_APPSEC: {}}  # type: NamespaceMetricType
+    metric_class = {"count": CountMetric, "gauge": GaugeMetric, "rate": RateMetric}
+
+    def _flush(self):
```

Comment:
`reset` might be a more appropriate name for this functionality, if I understand correctly what it's doing.

Reply:
I set _flush name because TelemetryWriter use the same terms to retrieve and reset the data